paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/PaymentSheetPlaygroundViewModel.kt

```diff
@@ -55,15 +56,20 @@ internal class PaymentSheetPlaygroundViewModel(
     val state = MutableStateFlow<PlaygroundState?>(null)
     val flowControllerState = MutableStateFlow<FlowControllerState?>(null)
 
+    val isWaitingToAutoRefresh = MutableStateFlow(false)
```

Comment:
What is this doing?

Reply:
It disables the buttons as soon as you change a setting, so before the debounced query begins.